### PR TITLE
Add sphinx jquery extension to fix read-the-docs search

### PR DESCRIPTION
**Pull request by bmhan12**
This PR:

- Fixes issue where search is broken with add-ons enabled in ReadTheDocs
  - Can try RTD search bar in develop documentation version to observe brokenness
- ReadTheDocs link to this branch: https://axom.readthedocs.io/en/bugfix-han12-rtd_search/
- Same as RAJA's LLNL/RAJA#1721

### CODE REVIEW

**Review by BradWhitlock — APPROVED** [review]
Looks good.

**Review by rhornung67 — APPROVED** [review]

File: src/conf.py [hunk 1]
```diff
@@ -57,7 +57,8 @@
     'sphinxcontrib.jquery',
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
-    'sphinx.ext.mathjax'
+    'sphinx.ext.mathjax',
+    'sphinxcontrib.jquery'
```
Comment (white238) [inline src/conf.py:61]: This is already on line 57

Comment (BradWhitlock) [reply src/conf.py:61]: D'oh - I was too reliant on diff highlighting.

Comment (bmhan12) [reply src/conf.py:61]: interesting... I'll play around with this a bit.

Comment (bmhan12) [reply src/conf.py]: Looks to be only "develop" branch does not have the search working.

Comment (white238) [reply src/conf.py]: Last time this happened, @kennyweiss helped by inspecting the html errors via the browser. This line specifically is what fixed it.

**Comment by bmhan12** [comment]
As @white238 pointed out, we already have `jquery` added.
Closing this PR.
I think the issue was related to do with my web browser caching. Not observing the error in a private window or by switching browsers.

File: src/conf.py (hunk 1 above)
Comment (bmhan12) [reply src/conf.py]: Scratch that, works when I switch browsers/use a private tab. Probably just need to clear cache.

**Pull request closed without merging**